electrumsv/gui/qt/main_window.py

```diff
@@ -1697,8 +1697,20 @@ def broadcast_tx():
                     self.payment_request = None
                 return
             else:
-                # wallet.set_transaction_state(tx.txid(), TxFlags.StateDispatched)
-                return self.network.broadcast_transaction_and_wait(tx)
+                result = self.network.broadcast_transaction_and_wait(tx)
+
+                # Success --> StateDispatched
+                if result == tx.txid():
+                    self.logger.debug(
+                        f'broadcast_tx in main_window successful '
+                        f'broadcast for: {result}')
+                    self.logger.debug(f'setting transaction to StateDispatched...')
+                    wallet.set_transaction_state(
```

Comment:
Get rid of debug logging and remove custom indentation style. We should go something like this if you must wrap. But I am not sure this wouldn't be less than 120 characters anyway.
```
wallet.set_transaction_state(tx.txid(),
     (TxFlags.StateDispatched | TxFlags.HasByteData))
```